packs/data/extinction-curse-bestiary.db/shoony-hierarch.json

```diff
@@ -308,17 +799,135 @@
+                    "value": ""
+                }
+            },
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.bestiary-ability-glossary-srd.fFu2sZz4KB01fVRc"
+                }
+            },
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Low-Light Vision",
+            "sort": 900000,
+            "type": "action"
+        },
+        {
+            "_id": "qjVkOEKOhI3t0v8m",
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>Your small, blunt snout and labyrinthine sinus system make you resistant to phenomena that assail the nose. When you roll a saving throw against inhaled threats (such as inhaled poisons) and olfactory effects (such as xulgath stench), you get the outcome one degree of success better than the result of your roll.</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "key": "Note",
+                        "predicate": {
+                            "any": [
+                                "inhaled",
+                                "olfactory"
+                            ]
+                        },
+                        "selector": "saving-throw",
+                        "text": "When you roll a saving throw against an olfactory or inhaled effect, you get the outcome one degree of success better than the result of your roll.",
+                        "title": "{item|name}"
+                    },
+                    {
+                        "adjustment": {
+                            "all": "one-degree-better"
+                        },
+                        "key": "AdjustDegreeOfSuccess",
+                        "predicate": {
+                            "any": [
+                                "olfactory",
+                                "inhaled"
+                            ]
+                        },
+                        "selector": "saving-throw",
+                        "type": "save"
+                    }
+                ],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                }
+            },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.QX6yAh18emUndCOF"
+                }
+            },
```

Comment:
```suggestion
```

Reply:
Fixed in [5f3c718](https://github.com/foundryvtt/pf2e/pull/2810/commits/5f3c718c6b78d33f4c335a7b9398895043044537)